src/components/core/button/toggle-button.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+
+const ToggleButton = ({ isActive = false, onClick = () => {}, children = '', label = '' }) => {
```

Comment:
I believe you can just leave it with no default value for `children`. Empty string should be okay but let's make it close to React tradition.